src/main/java/net/openhft/chronicle/wire/JSONWire.java

```diff
@@ -619,12 +621,21 @@ public JSONValueOut typePrefix(@NotNull CharSequence typeName) {
             if (useTypes) {
                 startBlock('{');
                 bytes.append("\"@");
-                bytes.append(typeName);
+                bytes.append(applyAsAlias(classLookup, typeName));
                 bytes.append("\":");
             }
             return this;
         }
 
+        private CharSequence applyAsAlias(ClassLookup classLookup, CharSequence typeName) {
+            // TODO use classLookup.applyAsAlias(typeName);
```

Comment:
Hi @peter-lawrey - there is still a TODO here

Reply:
Indeed, once the changes in Core have propagated this method can be used instead.